dockerfile

```diff
@@ -0,0 +1,18 @@
+# We use an official Python runtime as a parent image
+FROM python:3.7
+
+# The environment variable ensures that the python output is set straight
+# to the terminal without buffering it first
+ENV PYTHONUNBUFFERED 1
+
+# Create root directory for our project in the container
+RUN mkdir /perf_dashboard
+
+# Set the working directory to /perf_dashboard
+WORKDIR /perf_dashboard
+
+# Copy the current directory contents into the container at /perf_dashboard
+ADD . /perf_dashboard/
+
+# Install any needed packages specified in requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
I also would add an `USER 65534` statement to run the dashboard as user nobody (I assume the dashboard has no special requirements.)

Reply:
I'd rather not add nobody. Actually, I haven't add any authentication part in this app. But it do have an admin account which user can create. That's why to some extend we can still add access controls on this app.

Reply:
this is referring to the UID running thr process not app authentication

Reply:
oh, if so, I can add nobody.